Composer/packages/lib/ui-shared/src/components/DisplayReadme/DisplayReadme.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { DialogFooter } from 'office-ui-fabric-react/lib/Dialog';
+import { PrimaryButton, DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { ScrollablePane } from 'office-ui-fabric-react/lib/ScrollablePane';
+import formatMessage from 'format-message';
+import ReactMarkdown from 'react-markdown';
+
+import { DialogTypes, DialogWrapper } from '../DialogWrapper';
+
+type DisplayReadmeProps = {
+  title: string;
+  readme: string;
+  hidden: boolean;
+  onBack?: () => void;
```

Comment:
what is this supposed to be used for?

Reply:
In some cases, there may be a OK button AND a back button - OK would dismiss, but back would potentially do other state reset.

Reply:
Not used so I have removed